apps/web/components/Templates/Filters.tsx

```diff
@@ -0,0 +1,39 @@
+import Favorites from "./Favorites";
+import FilterList from "./FilterList";
+
+export default function Filters() {
+  const frameworks = [
```

Comment:
I think Subho has created a data structure for this before, it should be probably exported from one central location.

@subhoghoshX I remember you did something similar before, somewhere

Reply:
I think you're talking about [this](https://github.com/AndrejJurkin/template-bhai/blob/main/apps/web/utils/template-data.json). Some parts of it might be reused but not all of it I guess.